teamcity/manifest.json

```diff
@@ -10,6 +10,10 @@
   "manifest_version": "1.0.0",
   "metric_prefix": "teamcity.",
   "name": "teamcity",
+  "process_signatures": [
+    "runnAll.sh start",
```

Comment:
This seems like something that would trigger a false positive, and I think it should be `runAll.sh`?

Reply:
You would change it to `runAll.sh` or simply remove this signature ?

Reply:
If we decide to keep it in, I would change it to `runAll.sh start`. Is there a better/more specific way to detect TeamCity 9.x?

Reply:
Ok, let's do it. It sounds right to me :)